test/settings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
What's this file about? Couldn't find something in the `CONTRIBUTING.md` file and this seems to only be used in test cases.